docs/mutation-testing.md

```diff
@@ -0,0 +1,109 @@
+# Mutation Testing
+
+This document describes how to run mutation testing locally to mimic the outcome of a PR, without the CI limitation it provides by timing out after 6 hours.
+[Here is the script](../contrib/tools/local-mutation-testing.sh) to run mutation locally running the mutants created by the changes between the current branch and develop. It does automatically all the steps explained below.
+
+From the root level of the stacks-core repository run
+```sh
+./contrib/tools/local-mutation-testing.sh
```

Comment:
Be sure to chmod this file so it is executable. looks like it's currently 0644. or, update command to `bash ./contrib/tools/local-mutation-testing.sh` etc